mappings/net/minecraft/resource/DefaultResourcePack.mapping

```diff
@@ -3,12 +3,15 @@ CLASS wk net/minecraft/resource/DefaultResourcePack
 	FIELD b RESOURCE_CLASS Ljava/lang/Class;
 	FIELD c namespaces Ljava/util/Set;
 	FIELD d LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD e typeToFileSystem Ljava/util/Map;
+	METHOD <init> ([Ljava/lang/String;)V
+		ARG 1 namespaces
 	METHOD a getIdentifiers (ILjava/lang/String;Ljava/nio/file/Path;Ljava/lang/String;Ljava/util/function/Predicate;)Ljava/util/Collection;
 		ARG 1 maxDepth
 		ARG 2 namespace
 		ARG 3 path
 		ARG 4 searchLocation
-		ARG 5 filter
+		ARG 5 fileNameFilter
```

Comment:
For `DefaultResourcePack`, the file name is the same as the identifier, so it's really an `identifierFilter` (it's also the same argument that gets passed to `ResourceManager.findResources`, which is not file-specific), and since the method is `getIdentifier`, `filter` is enough.

Reply:
Discussion below.